esx_service/vmci/vmci_client.c

```diff
@@ -161,9 +165,10 @@ dummy_get_reply(be_sock_id *id, be_request *r, be_answer* a)
 static be_sock_status
 vsock_init(be_sock_id *id, int cid, int port)
```

Comment:
Ah, the old code was *reusing* the same port for outbound and inbound, with no related comments :-(. Yes, the change makes it better.